### PR TITLE
docs: Mark parts to skip for no calico.

### PR DESCRIPTION
**Pull request by lwr20**
Make it obvious which parts can be skipped if user does not want to use Calico 
(for example, if they are setting up the simplest possible test cluster).

### CODE REVIEW

File: Documentation/deploy-master.md [hunk 1]
```diff
@@ -395,7 +397,9 @@ spec:
         - "--http=127.0.0.1:4040"
 ```
 
-### Set Up the CNI config
+### Set Up the CNI config (optional)
+This step can be skipped if you do not wish to provide network policy to your cluster using Calico.
```
Comment (robszumski) [inline Documentation/deploy-master.md]: newline between header and text for consistency

File: Documentation/deploy-master.md [hunk 2]
```diff
@@ -460,7 +464,8 @@ $ sudo systemctl enable kubelet
 Created symlink from /etc/systemd/system/multi-user.target.wants/kubelet.service to /etc/systemd/system/kubelet.service.
 ```
 
-### Start Calico
+### Start Calico (optional)
+This step can be skipped if you do not wish to provide network policy to your cluster using Calico.
```
Comment (robszumski) [inline Documentation/deploy-master.md]: newline between header and text for consistency

**Comment by robszumski** [comment]
LGTM other than minor nits

**Comment by lwr20** [comment]
Thanks for the review comments - I've added the requested newlines.

**Comment by aaronlevy** [comment]
lgtm

**Pull request merged**